scala-rl-book/src/main/scala/io/samritchie/rl/book/Chapter3.scala

```diff
@@ -38,14 +38,14 @@ object Chapter3 {
     function instead here to get this working, to check that the maximum delta
     is less than epsilon.
     */
-  def valueFunctionConverged[Obs](
-      l: StateValueFn[Obs],
-      r: StateValueFn[Obs]
+  def valueFunctionConverged[Obs, T: ToDouble](
```

Comment:
this is much more complicated now, with all of the types, since we're coming in at the fully generic level. I need another layer of abstraction, to lock down these obvious types.